config/cookies.json

```diff
@@ -1 +1,5 @@
-
+[{
+   "name": "SSESS...42833",
+   "value": "DdPx4...7%7py",
+   "domain": ".feature.commondesign-unocha-org.ahconu.org"
```

Comment:
How is this going to work? Does it need to be changed depending on the contents of the .env file?

Reply:
Ah no, this is an example only for reference. The cookies file does not need to be edited; the Jenkins job will do that as it gets session cookies.